packages/filesystem/src/browser/file-tree/file-tree-model.ts

```diff
@@ -109,4 +111,51 @@ export class FileTreeModel extends TreeModel implements LocationService {
         return true;
     }
 
+    upload(destination: string, file: File, fileName: string) {
+        const fileToCreate = destination + "/" + fileName;
+        this.fileSystem.exists(fileToCreate).then(exist => {
```

Comment:
doesn't seem like `exist` is necessary or used here

Reply:
Code modified